python/requirements.in

```diff
@@ -1,23 +1,24 @@
 black==23.1.0
 boto3>=1.28.1
+cached-property==1.5.2
 contextily==1.3.0
 folium==0.14.0
 geopandas==0.12.2
 ipykernel==6.22.0
 Jinja2==3.1.2
 mapclassify==2.5.0
 matplotlib==3.7.1
+openpyxl==3.1.2
 pandas==1.5.3
 psycopg2-binary==2.9.5
 pytest==7.2.2
 pytest-cov==4.0.0
 python-dotenv==0.20.0
-python-geosupport @ git+https://github.com/NYCPlanning/python-geosupport.git@3b8df06
```

Comment:
@fvankrieken 
So we do need this for `facdb`, but you mentioned it was causing an issue? When I pip install, I just get the following `WARNING: Did not find branch or tag '3b8df06', assuming revision or ref.` which is what we'd expect. 

I'd just pinned it to this commit [here](https://github.com/NYCPlanning/python-geosupport/commit/3b8df06da5800ff38d0b75bc0b27181fbc919c3b). Maybe we just unpin it, and use the latest?

Reply:
Re our convo, I was wrong about the install, BUT we've decided to unpin (and just rely on whatevers installed in docker image) and see if FacDB builds